devicetypes/smartthings/zwave-basic-heat-alarm.src/zwave-basic-heat-alarm.groovy

```diff
@@ -0,0 +1,165 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Z-Wave Basic Heat Alarm", namespace: "smartthings", author: "SmartThings") {
+		capability "Temperature Alarm"
+		capability "Sensor"
+		capability "Battery"
+		capability "Health Check"
+
+		//zw:S type:0701 mfr:026F prod:0001 model:0002 ver:1.07 zwv:4.24 lib:03 cc:5E,86,72,5A,73,80,71,85,59,84 role:06 ff:8C01 ui:8C01
+		fingerprint mfr: "026F ", prod: "0001", model: "0002", deviceJoinName: "FireAngel Thermistek ZHT-630 Heat Alarm/Detector"
```

Comment:
Since this is a generic device type, can we also add a generic device fingerprint based on device type and command classes (deviceId and inClusters)?

Reply:
The may be one problem with that. Same classes are used by FireAngel Smoke sensor. Only diffrence is model number 0001 instead of 0002.

Reply:
bummer